scripts/servers.py

```diff
@@ -380,20 +380,23 @@ def create_managed_web_browser(
     Returns:
         context manager|None. The context manager to a web browser window, or
         None if the current operating system does not support web browsers.
+
+    Raises:
+        Exception. Could not attempt to launch the web browser (When Operating
```

Comment:
"Could not launch web browser (this happens when the operating system cannot be identified)."

Reply:
Fixed